.github/workflows/ci.yml

```diff
@@ -255,7 +255,7 @@ jobs:
     if: needs.detect-changes.outputs.java-code-changes == 'true'
     name: "General / Unit - Back End"
     needs: [ detect-changes, setup-unit-tests ]
-    timeout-minutes: 10
+    timeout-minutes: 25
```

Comment:
Wow, that is really a tremendous increase in runtime indeed. Thanks for looking into it after the fact.